modules/tinymce/src/core/main/ts/content/SetContentImpl.ts

```diff
@@ -99,5 +96,5 @@ export const setContentInternal = (editor: Editor, content: Content, args: SetCo
     } else {
       return setContentString(editor, body, content, args);
     }
-  }).getOr({ content, html: args.content });
+  }).getOr({ content, html: isTreeNode(args.content) ? '' : args.content });
```

Comment:
This here is just a case for where the editor has been killed, so it can be anything so I just made it be the same as it was previously. I also didn't see the point of serializing the tree here but I'm happy to if needed.